libraries/core/src/test/java/com/paritytrading/philadelphia/FIXInitiatorTest.java

```diff
@@ -44,31 +45,29 @@ class FIXInitiatorTest {
     private FIXMessages  initiatorMessages;
     private TestMessages acceptorMessages;
 
-    private FIXConnectionStatus initiatorStatus;
+    private FIXConnectionStatus<INITIATOR_CHAN> initiatorStatus;
 
-    private FIXConnection  initiator;
+    private FIXConnection<INITIATOR_CHAN> initiator;
     private TestConnection acceptor;
 
-    @BeforeEach
-    void setUp() throws IOException {
-        ServerSocketChannel acceptorServerChannel = ServerSocketChannel.open();
-        acceptorServerChannel.bind(null);
-
-        SocketChannel initiatorChannel = SocketChannel.open(acceptorServerChannel.getLocalAddress());
-        initiatorChannel.configureBlocking(false);
-
-        SocketChannel acceptorChannel = acceptorServerChannel.accept();
-        acceptorChannel.configureBlocking(false);
+    protected final class Channels {
+        public INITIATOR_CHAN initiator;
+        public ByteChannel acceptor;
+    }
```

Comment:
This class could be in the package-private scope like other non-private members in this class.